Makefile

```diff
@@ -177,6 +180,10 @@ test-go-integration: gotestsum ## run go integration tests.
 test-go-e2e: gotestsum ## run go e2e tests.
 	$(GOTESTSUM) $(TEST_ARGS) ./test/e2e/... -run E2E
 
+test-k6-performance: ## run k6 performance tests.
```

Comment:
add a `.PHONY` directive above

Reply:
add `xk6 k6` as dependencies to this target

Reply:
added in L169